packages/react-components/src/foundations/radius.css

```diff
@@ -0,0 +1,6 @@
+:root {
+  --spacings-border-radius-none: 0;
+  --spacings-border-radius-xs: 4px;
+  --spacings-border-radius-sm: 6px;
+  --spacings-border-radius-md: 8px;
```

Comment:
We already have a `spacing` keyword used in a singular form. Maybe we should use the singular form here as well, for consistency? WDYT? @marcinsawicki @vladko-uxds

Reply:
I agree with that, but we need to unify the naming with our documentation. CC: @vladko-uxds